boards/seeeduino_xiao/board.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C)  2021 Franz Freitag, Justus Krebs, Nick Weiler
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_seeeduino_xiao
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the Seeeduino XIAO board
+ *
+ * @author      Franz Freitag <franz.freitag@st.ovgu.de>
+ * @author	Justus Krebs <justus.krebs@st.ovgu.de>
+ * @author	Nick Weiler <nick.weiler@st.ovgu.de>
+ * @author	Benjamin Valentin <benpicco@googlemail.com>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "board.h"
+#include "mtd.h"
+#include "mtd_spi_nor.h"
+#include "periph/gpio.h"
+#include "periph/spi.h"
+#include "timex.h"
+
+#ifdef MODULE_MTD
+/* GD25Q32C */
```

Comment:
Is this the chip that's used?
Have you tried `tests/mtd_raw`?

Reply:
After some research we decided that we delete the spi nor flash because the board does not support this.

Reply:
As shown in the [https://files.seeedstudio.com/wiki/Seeeduino-XIAO/res/Seeeduino-XIAO-v1.0-SCH-191112.pdf](url)